barretenberg/cpp/src/barretenberg/bb/readme.md

```diff
@@ -31,7 +31,7 @@ This binary will take as input ACIR and witness values described in the IR to cr
 3. Install the version of `bb` compatible with your Noir version; with Noir v0.32.0 for example:
 
     ```bash
-    bbup -v 0.46.1
+    bbup -v 0.47.1
```

Comment:
did the version compatible with 0.32 change?

Reply:
should line 31 also be changed?

Reply:
Thanks for the catch! Updated to Noir v0.34.0 <> bb v0.55.0 directly in one-go.